app/cafe.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+from app.errors import NotVaccinatedError, OutdatedVaccineError
+from app.errors import NotWearingMaskError

```

Comment:
Use one `import` to import all errors. Use such style if there are a lot of classes
```
from app.errors import (
     NotVaccinatedError,
     OutdatedVaccineError,
     NotWearingMaskError
)
```